commit f6fe7143e47c32ce08620238f63887377e72533a
Merge: dde4b40 bae1e42
Author: Bernhard Kubicek <kubicek@gmx.at>
Date:   Fri Dec 9 06:37:42 2011 -0800

    Merge pull request #29 from k-eex/Marlin_v1
    
    Fix M201 not saving settings in all places, so the changes did not get written into EEPROM

diff --git a/Marlin/Marlin.pde b/Marlin/Marlin.pde
index 05924b9..538920d 100644
--- a/Marlin/Marlin.pde
+++ b/Marlin/Marlin.pde
@@ -989,7 +989,11 @@ FORCE_INLINE void process_commands()
     case 201: // M201
       for(int8_t i=0; i < NUM_AXIS; i++) 
       {
-        if(code_seen(axis_codes[i])) axis_steps_per_sqr_second[i] = code_value() * axis_steps_per_unit[i];
+        if(code_seen(axis_codes[i]))
+        {
+          max_acceleration_units_per_sq_second[i] = code_value();
+          axis_steps_per_sqr_second[i] = code_value() * axis_steps_per_unit[i];
+        }
       }
       break;
     #if 0 // Not used for Sprinter/grbl gen6
